src/Microsoft.VisualStudio.Editors/SettingsDesigner/SettingsValueSerializer.vb

```diff
@@ -11,9 +11,12 @@ Namespace Microsoft.VisualStudio.Editors.SettingsDesigner
     ''' a culture info.
     ''' </summary>
     ''' <remarks></remarks>
-    Friend Class SettingsValueSerializer
+    Friend NotInheritable Class SettingsValueSerializer
```

Comment:
I assume this can't be shared because of one of the InternalsVisibleTo attributes? If so, how does the existing changes not break that scenario already?

Reply:
VB doesn't have an equivalent of "static" classes. They have modules, but that results in VB importing those methods for usage without class prefix everywhere.

Reply:
Thank you, I suspect this won't be the last time I forget that :)